.github/workflows/_reusable_build.yml

```diff
@@ -17,30 +17,17 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
-      - uses: Keeper-Security/ksm-action@v1
+      - name: Configure Maven Settings
+        uses: bonitasoft/maven-settings-action@v1
```

Comment:
Move the `maven-settings-action` after `actions/setup-java` because setup-java will override the settings.xml otherwise.